test/jest-setup-after-env.ts

```diff
@@ -4,3 +4,10 @@ expect.extend(matchers)
 // A default max-timeout of 90 seconds is allowed
 // per test we should aim to bring this down though
 jest.setTimeout((process.platform === 'win32' ? 180 : 90) * 1000)
+
+// Silence logs in CI
```

Comment:
Doesn't jest have a feature for this built in?

Reply:
It would be cool if it had. There is something monkey patching `console.log` but I wasn't able to find it.

Reply:
There are a few config options for jest, but it seems related to jest itself and not the code that is executed.
And we want `require("console").log` to be visible, which wouldn't be the case if we silenced it with `--silent` or something.

Reply:
But I haven't dug that deep here.

Reply:
I think `jest --silent` https://jestjs.io/docs/cli#--silent

Reply:
> There are a few config options for jest, but it seems related to jest itself and not the code that is executed.
And we want require("console").log to be visible, which wouldn't be the case if we silenced it with --silent or something.

But I can try if that will work.

Reply:
Cool, it seems that it lets logs through if there is an error, and it also let's trough `require("console").log`.